src/graphql/root/subscription/my-updates.ts

```diff
@@ -250,8 +271,8 @@ const MeSubscription = {
       const priceData = {
         timestamp: pricePerSat.timestamp,
         displayCurrency,
-        centsPerSat: usdMajorToMinorUnit(pricePerSat.price),
```

Comment:
Did this used to be major unit coming out of `pricePerSat` before?

Reply:
price service returns price per sat in major unit.